### PR TITLE
BOAC-35, code style fixes: import order, etc.

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/BOAC-35

Code style fixes and removed corresponding codes from `flake8 --ignore` list.

A question for the team: Where do we stand on [trailing commas in Python data structures](https://coderwall.com/p/4fvcaa/trailing-comma-in-python-data-structures) (as enforced by `flake8-commas`)? 

Also: https://wiki.python.org/moin/TupleSyntax (see "...essential element here is the trailing comma...")

### CODE REVIEW

File: boac/api/status_controller.py [hunk 1]
```diff
@@ -11,10 +10,10 @@ def app_status():
         'is_authenticated': current_user.is_authenticated,
         'is_active': current_user.is_active,
         'is_anonymous': current_user.is_anonymous,
-        'uid': uid
+        'uid': uid,
```
Comment (johncrossman) [inline boac/api/status_controller.py]: Trailing comma looks out of place but there are some legit considerations:
* https://wiki.python.org/moin/TupleSyntax (see "...essential element here is the trailing comma...")
* See bug described in https://coderwall.com/p/4fvcaa/trailing-comma-in-python-data-structures

Comment (raydavis) [reply boac/api/status_controller.py]: I could get used to it, now that I know it's acceptable.

File: tox.ini [hunk 2]
```diff
@@ -11,24 +11,27 @@ skipsdist = True
 [testenv]
 whitelist_externals = *
 
-[testenv:pytest]
+[testenv:test]
 commands =
     pytest {posargs}
 
-[testenv:eslint]
+[testenv:lint-js]
```
Comment (johncrossman) [inline tox.ini]: Fyi, renamed targets. Genericized! Suggestions welcome.

File: boac/api/status_controller.py (hunk 1 above)
Comment (paulkerschen) [reply boac/api/status_controller.py]: The trailing commas are FREAKING ME OUT but I'm up for trying new thing.

FWIW here's what PEP8 has to say:
https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas

**Pull request merged**